dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -292,11 +292,22 @@ function get_free_disk_bytes {
 
 function get_partition_table_type {
     local disk_device="$1"
+    local table_type
 
     # blkid doesn't work for dasd partitions, so on s390 parted is used
     # to detect the partition type (bsc#1209247)
     if [[ "$(uname -m)" =~ s390 ]];then
-        parted --script --machine "${disk_device}" print | grep "^${disk_device}" | cut -d":" -f6
+        table_type=$(
+            parted --script --machine "${disk_device}" print |\
+                grep "^${disk_device}" | cut -d":" -f6
+        )
+        if [ "${table_type}" = "msdos" ];then
+            # make sure to set a consistent table name for the
+            # DOS table type. Some tools reports it as either
```

Comment:
> Some tools reports it as either

I'd hope that at least `parted` reports it as `msdos` consistently

Reply:
yes it does and I agree the comment could be misleading. I changed the comment text to be more clear